ykrt/pass_finder/cargo_run.py

```diff
@@ -1,22 +1,37 @@
 import os, subprocess, time
 
 def run_test(yk_path,env=None, n=1) :
+    if env is None:
+        env = os.environ.copy()
+    env['PATH'] = f"{yk_path}/bin:{env['PATH']}"
+    env['YK_BUILD_TYPE'] = 'debug'
     prelink_passes = (env or os.environ).get('PRELINK_PASSES', 'Not Set')
     print(f"\033[93mPRELINK_PASSES: {prelink_passes}\033[0m")
-    os.chdir("/home/shreei/research/yk_pv")
-    #subprocess.run(["cargo build"], shell=True, env=env or os.environ)
-    times = []
-    for i in range(n):
-        before = time.time() 
-        c = subprocess.run(["timeout 60 cargo test"], shell=True, env=env or os.environ) 
-        elapsed = time.time() - before
-        times.append(elapsed)
-    mean_time = sum(times) / len(times)
+    os.chdir(yk_path)
+    r = subprocess.run(["timeout 50 cargo test"], shell=True, env=env or os.environ)
+    # assert r.returncode == 0
+    if r.returncode == 0:
+        os.chdir("/home/shreei/research/yklua")
```

Comment:
Not convinced that this will work on my machine :p

Reply:
This has been addressed.

Reply:
I don't see anything in https://github.com/ykjit/yk/commit/7e2186b965e3a4b2fff39a563ffcf1a3cc36c8db about paths with `shreei`?

Reply:
I couldn't find which commit I solved it it, but there is no hardcoded path anymore.

Reply:
If you can't find the commit, how do we know?

Reply:
Sorry, I found the commit: b179d7cd38f5c06aee45e10b01b907aac31ac5e3